src/routes/api/tripRoute.js

```diff
@@ -239,4 +240,61 @@ tripRouter.post('/twoWay',
  *
  */
 
+tripRouter
+  .put(
+    '/edit/:id',
```

Comment:
Can you use `tripRequestId` instead of `id`, so that we may all reflect on the same ID name?

Reply:
Done.